traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -164,6 +164,7 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.1, http.MethodGet, `users/{id}/deliveryservices/?(\.json)?$`, user.GetDSes, auth.PrivLevelReadOnly, Authenticated, nil},
 		{1.1, http.MethodGet, `user/{id}/deliveryservices/available/?(\.json)?$`, user.GetAvailableDSes, auth.PrivLevelReadOnly, Authenticated, nil},
 		{1.1, http.MethodPost, `user/login/?$`, login.LoginHandler(d.DB, d.Config), 0, NoAuth, nil},
+		{1.4, http.MethodPost, `user/login/oauth$`, login.OauthLoginHandler(d.DB, d.Config), 0, NoAuth, nil},
```

Comment:
most other endpoints support an optional trailing slash. It's probably not a huge deal, but it's also only a 2-character change to support it.

Reply:
done